### PR TITLE
LPS-178761 | Test Automation | Master

### PR DESCRIPTION
**Pull request by davisaints**
**JIRA Ticket: [LPS-178761](https://issues.liferay.com/browse/LPS-178761)**

### CODE REVIEW

**Comment by davisaints** [comment]
ci:test:sf

**Comment by davisaints** [comment]
ci:test:sf

**Comment by davisaints** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/Object.testcase [hunk 1]
```diff
@@ -510,6 +510,73 @@ definition {
...
+				accountEntryName = "Accounts Name",
+				accountEntryType = "Business");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectAdmin.addNewObject(
+				fieldLabelName = "Custom Object 178761",
+				pluralLabelName = "Custom Objects 178761");
+
+			CreateObject.saveObject();
```
Comment (andreefarias) [inline modules/apps/object/object-test/src/testFunctional/tests/Object.testcase:541]: Could you replace these steps to do via API?

```
ObjectAdmin.addObjectViaAPI(
	labelName = "Custom Object 178761",
	objectName = "CustomObject178761",
	pluralLabelName = "Custom Objects 178761");
```

**Review by andreefarias — CHANGES_REQUESTED** [review]
Hey @davisaints, good job with this testcase. It's good for me, I asked only to change the steps via UI to API, this way the testcase will be more quickly. 

Thanks!

File: modules/apps/object/object-test/src/testFunctional/tests/Object.testcase (hunk 1 above)
Comment (davisaints) [reply modules/apps/object/object-test/src/testFunctional/tests/Object.testcase:541]: For some reason the language change in the virtual instance prevented an object from being created by the API, so I need to do it through the UI.

**Review by andreefarias — APPROVED** [review]

**Comment by davisaints** [comment]
ci:test:sf

**Comment by andreefarias** [comment]
ci:forward

**Pull request closed without merging**